docs/pipeline_terraform.md

```diff
@@ -0,0 +1,79 @@
+## Automation for Terraform VFXT Pipeline:
```

Comment:
Should this doc be published in the public-facing GitHub? This seems useful only to internal users since external users cannot run our predefined pipelines, and this document deep-dives into the internals of how these pipelines run. I'm inclined to keep this in an internal doc repository instead.

If you want to keep this public-facing, then some potential reorganization should be done. Additionally, we should have our Doc folks review this before publication.

Reply:
I'll remove from this. I will add and link it to you in teams.